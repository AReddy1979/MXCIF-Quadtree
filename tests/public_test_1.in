INIT_QUADTREE(8)
DISPLAY()
LIST_RECTANGLES()
CREATE_RECTANGLE(DEVO,5,7,2,12)
INSERT(311)
TRACE ON
SEARCH_POINT(25,23)
DELETE_RECTANGLE(RUSH)
DELETE_POINT(15,13)
TOUCH(PRINCE)
WITHIN(CCR,10)
