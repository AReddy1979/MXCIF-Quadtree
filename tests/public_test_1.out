1 8
2
3
4 DEVO 5 7 2 12
7 311
5 25 23
8A RUSH
8B 15 13
10 PRINCE
11 CCR 10
